### PR TITLE
Inoue pathology detail dict 345

### PR DESCRIPTION
**Pull request by bullenc**

### CODE REVIEW

File: src/gdcdictionary/schemas/pathology_detail.yaml [hunk 1]
```diff
@@ -236,6 +236,14 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  days_to_pathology_detail:
+    description: >-
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/pathology_detail.yaml:240]: should we put the description in _terms file? ditto `micrometastasis_present` below

**Review by wteouchicago — APPROVED** [review]

File: src/gdcdictionary/schemas/pathology_detail.yaml (hunk 1 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/pathology_detail.yaml:240]: Yeah we need to flesh out a process for what to do in terms definition adding when we do not have a code in a terms database (NCIt, caDSR etc) but still want to offer a description, especially for properties. I think for now we're ok but if/when the EVS team hopefully generates a code for this we can formally add to terms file with associated definition ID.

**Review by clsackett — APPROVED** [review]

File: src/gdcdictionary/schemas/pathology_detail.yaml [hunk 2]
```diff
@@ -350,6 +400,25 @@ properties:
...
+  histologic_progression_type:
+    $ref:
+      - "_terms.yaml#/histologic_progression_type/common"
+    enum:
+      - Anaplastic
+      - Poorly differentiated
+      - Unknown
+      - Not Reported
+    enumDef:
+      Poorly differentiated:
```
Comment (wrkhenddher) [inline src/gdcdictionary/schemas/pathology_detail.yaml:412]: Isn't this missing `Anaplastic` ?

File: src/gdcdictionary/schemas/pathology_detail.yaml [hunk 3]
```diff
@@ -982,13 +1080,6 @@ properties:
         - Base
         - Middle
 
-  tumor_measurement_method:
-    $ref:
-      - "_terms.yaml#/tumor_measurement_method/common"
-    enum:
-      - Echographic
-      - Pathologic
-
```
Comment (wrkhenddher) [inline src/gdcdictionary/schemas/pathology_detail.yaml:991]: Are these deprecated?
or just removed outright ?

Comment (wrkhenddher) [reply src/gdcdictionary/schemas/pathology_detail.yaml:991]: and what about it in `_terms.yaml`?

**Review by wrkhenddher — APPROVED** [review]

File: src/gdcdictionary/schemas/pathology_detail.yaml (hunk 3 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/pathology_detail.yaml:991]: We are removing these outright- they were not used and are redundant with another property.

**Pull request merged**